src/python/lib/ensembl/compara/utils/cigar.py

```diff
@@ -14,9 +14,57 @@
 # limitations under the License.
 """Collection of utils methods targeted to cigar line manipulation."""
 
-__all__ = ['aligned_seq_to_cigar', 'alignment_to_seq_coordinate', 'get_cigar_array']
-
-from typing import List, Tuple
+__all__ = ['aligned_seq_to_cigar', 'alignment_to_seq_coordinate',
+           'get_cigar_array', 'alignment_to_seq_region']
+
+from typing import List, Tuple, Optional
+
+def alignment_to_seq_region(cigar: str, start: int, end: int) -> Optional[Tuple[int,int]]:
```

Comment:
```suggestion
def alignment_to_seq_region(cigar: str, start: int, end: int) -> Optional[tuple]:
```
I wonder if this would be okay (going by Travis' complaints)? I can't see why you would need to use `Tuple[]` rather than python's `tuple`?

Reply:
in fact mypy is complaining if I do not specify this because it is not specific enough,

Reply:
I am confused because you are using a `tuple` rather than a `Tuple[]`. I expected the error message that mypy is producing is based on mypy implicitly deriving `Tuple[]` from the `return` code - but you are not really returning a `Tuple[]`, but a `tuple`, which is causing the issue. How does the error message change when you use my suggestion?